rpi_backlight.py

```diff
@@ -45,38 +46,53 @@ def _set_value(name: str, value: Any) -> None:
             _perm_denied()
 
 
+def set_brightness_value(value: int) -> None:
```

Comment:
Still not convinced about this, I already wrote a comment. This should not be part of the public API, I#d rather remove it altogether.

Reply:
I don't think helper functions that remove common repeated lines of code are bad
How can we remove it from public API without getting rid of it?

Reply:
fixed